2016-08/august-05.md

```diff
@@ -0,0 +1,73 @@
+## July 29 ([discuss](https://github.com/webpack/meeting-notes/pull/8))
+
+### Attendees
+
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Juho Vepsäläinen](http://github.com/bebraw)
+* [Johannes Ewald](http://github.com/jhnns)
+* [Jesus Rodriguez](https://github.com/foxandxss)
+
+#### Attendee Notes:
+Tobias is going to be on vacation for a couple weeks and thus wasn't present for this meeting.
+
+### Agenda
+
+#### Documentation Updates
+
+* Sean has [merged](https://github.com/webpack/webpack.io/pull/59) our first [new documentation](https://github.com/webpack/webpack.io/blob/develop/content/concepts/index.md) into webpack/webpack.io!!!! Share it around, and provide feedback. It is still in a beta state as there is some link cleanups. But he wanted to provide a great introduction to people who haven't already had one. 
+
+* We are still on the hunt for the `webpack.io` domain name. We see its registered to a gentleman in China, and would love any Mandarin speakers to help us draft an email to reach out to him for domain purchase as the first few emails have been unsuccessful!! [Marco Pai](https://github.com/MarcoPai) has some slight activity on github. So we aren't giving up hope just yet.
+
+* Currently the `webpack/webpack.io` is a statically generated site, so Juho is going to be working to add React and JS for Dynamic usage in other portions of the page. 
+
+* Documentation Structure has been finalized!!!! Priority number one is now creating the content and merging it into develop! The styling for the page is still a work in progress, however that is our last priority before we "ship the docs". 
+
+* For future we may have Johannes with the help of other UI Designers to create personas that help us and contributors create appropriate language for documentation. Agreed that it was a very-nice-to-have but also a great addition if we ever had time. 
+
+* Would love to have folks work on a SchemaStore schema for the webpack configuration. This we could use to autogenerate some documentation like the API: Configuration page. Also could provide IDE support for people almost out of the box. Need to look into how to do this when we tackle that documentation. 
+
+* It was brought up that we need to create a feasible action plan that we can share with the Dan Abramov's, Kent Dodds', and Patrick Stapleton's of the JS community to pass along and promote. It is in everyones best interest to work together to create documentation and having a strategy to accomplish this is vital. We spent time listing out the immediate MVP documents that would the team agreed could allow us to life the webpack 2.x beta status. 
```

Comment:
👋  😄 
I'm not 100% certain I understand what this bullet point is trying to say. Happy to promote Webpack stuff :)